optaweb-employee-rostering-standalone/pom.xml

```diff
@@ -17,7 +17,7 @@
   <properties>
     <application.port>8180</application.port>
     <frontend.project.name>optaweb-employee-rostering-frontend</frontend.project.name>
-    <version.cypress.docker>3.6.0</version.cypress.docker>
+    <version.cypress.docker>4.12.1</version.cypress.docker>
```

Comment:
This should always be in sync with the version that we have in package.json but I don't know how to enforce that.

Reply:
One could create a program to verify that (either via a maven plugin OR node script), but I don't think one exists (there is one for maintaining project version between package.json and pom.xml, but not dependency version: https://www.npmjs.com/package/sync-pom-version-to-package)